test/fixtures/lightscript/arrow-methods/class-malformed-async-skinny-arrow/actual.js

```diff
@@ -0,0 +1,3 @@
+class X {
+  async method() -> await a
```

Comment:
....which of course should not throw `Can't use async with lightscript arrows. (2:2)` anymore, but should throw something. Chrome doesn't parse `async get foo() {}` or `get async foo() {}`. 

Maybe `Getters cannot be async` ?